README.md

```diff
@@ -50,6 +50,31 @@ These examples use an avro file available for download [here](https://github.com
 $ wget https://github.com/databricks/spark-avro/raw/master/src/test/resources/episodes.avro
 ```
 
+## Supported types
+As of now, every avro type with the exception of complex unions is supported. To be more specific, we use the following mapping from avro types to SparkSQL types:
+
+```
+boolean -> BooleanType
+int -> IntegerType
+long -> LongType
+float -> FloatType
+double -> DoubleType
+bytes -> BinaryType
+string -> StringType
+record -> StructType
+enum -> StringType
+array -> ArrayType
+map -> MapType
+fixed -> BinaryType
+```
+
+As for unions, we only support three kinds of unions:
+1) union(int, long)
+2) union(float, double)
+3) union(something, null), where something is one of the avro types mentioned above, including two types of unions.
```

Comment:
That would be left, right, righter, righterer, etc., I think, but it should work.